Pod/Tests/TestStash.swift

```diff
@@ -0,0 +1,43 @@
+import UIKit
+import XCTest
```

Comment:
I don't really have a preference. This is just easier to get started with seeing as `Quick` is not apart of the template but we could just update it and use `Quick` for all the things :sunglasses: 

What do you think @RamonGilabert, go `Quick` or go home? :grin: